src/components/MobileSidebar/MobileSidebar.css

```diff
@@ -0,0 +1,92 @@
+.sidebar-bg {
+  width: 100vw;
+  z-index: 9;
+  visibility: hidden;
+  opacity: 0;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: var(--modal-bg-100);
+  overflow: hidden;
+  transition: all 0.3s cubic-bezier(0.25, 0.46, 0.43, 0.96);
```

Comment:
`transition: all;`  is [not recommended](https://www.pno.dev/articles/dont-use-the-all-keyword-in-css-transitions/#:~:text=CSS%20animations%20and%20transitions%20are,when%20done%20the%20wrong%20way.).

Reply:
done